src/test/java/com/google/devtools/build/lib/rules/objc/AppleToolchainSelectionTest.java

```diff
@@ -41,12 +41,12 @@ public void testToolchainSelectionDefault() throws Exception {
 
     assertThat(cppConfig.getRuleProvidingCcToolchainProvider().toString())
         .isEqualTo("//tools/osx/crosstool:crosstool");
-    assertThat(cppConfig.getTransformedCpuFromOptions()).isEqualTo("ios_x86_64");
+    assertThat(cppConfig.getTransformedCpuFromOptions()).isEqualTo("darwin_x86_64");
```

Comment:
This is the only test I intentionally changed the behavior of, I believe that we want to now test the default is macOS (and therefore is this default arch)